### ISSUE TITLE
VSCode launch.json templates "runToMain" deprecated

### ISSUE THREAD

**Issue by jamd315** [raspberrypi/pico-examples#262]
In the templates launch-raspberrypi-swd.json and launch-remote-openocd.json under /ide/vscode, both use the deprecated debug attribute `runToMain: true`, which causes a warning to display in VSCode.  See also [cortex-debug debug attributes](https://github.com/Marus/cortex-debug/blob/master/debug_attributes.md)

![Screenshot from 2022-08-31 11-15-28](https://user-images.githubusercontent.com/6594497/187732528-db888452-b24d-426e-bdba-251ddd82fcc5.png)

**Comment by lurch** [comment]
ping @JamesH65

**Comment by JamesH65** [comment]
Yes, definitely a valid issue. Not sure if I can add the new command, which would be backwards compatible, or need to replace the runtomain completely, which is no so backwards compatible.
